samples/Rholang/hello_world_again.rho

```diff
@@ -0,0 +1,17 @@
+// https://github.com/rchain/rchain/tree/master/rholang/examples (MIT License)
+
+// A simple HelloWorld contract demonstrating a contract being rerun
```

Comment:
Linguist requires the use of real-world actual use samples in order to train the classifier. "Hello world" is _never_ real world code 😉. Please remove this sample.